ksql-cli/src/main/java/io/confluent/ksql/cli/console/cmd/CliCmdUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License; you may not use this file
+ * except in compliance with the License.  You may obtain a copy of the License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.cli.console.cmd;
+
+import io.confluent.ksql.util.KsqlException;
+import java.util.Collection;
+import java.util.function.Supplier;
+
+final class CliCmdUtil {
+
+  private CliCmdUtil() {
+  }
+
+  static void ensureArgCountBounds(
+      final Collection<String> args,
+      final int minArgCount,
+      final int maxArgCount,
+      final Supplier<String> helpSupplier
```

Comment:
a quick scan through this PR shows that we always just pass in `() -> literal`. Why make this a Supplier?

Reply:
Probably an artefact of the refactoring I did. I was originally a method. I'll take a look...

Reply:
Sorted.